src/main/java/org/team1540/bunnybotTank2023/commands/drivetrain/TankdriveCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.team1540.bunnybotTank2023.commands.drivetrain;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+
+import org.team1540.bunnybotTank2023.Constants;
+
+public class TankdriveCommand extends CommandBase {
+    private final Drivetrain drivetrain;
+
+    private final CommandXboxController xBox;
+
+    public TankdriveCommand(Drivetrain subsystem, CommandXboxController xBox) {
```

Comment:
I would name these parameters better, since `subsystem` doesn't tell me anything about what I'm passing. `xBox` is also unclear, whereas calling it something like `controller` tells me what you are passing and what it does.